proposals/0279-screen-manager-subscribe-buttons.md

```diff
@@ -34,16 +33,18 @@ public interface OnButtonListener {
     void onError(String info);
 }
 
-public void subscribeButton(ButtonName buttonName, OnButtonListener listener);
-public void unsubscribeButtonListener(OnButtonListener listener);
+public boolean addButtonListener(ButtonName buttonName, OnButtonListener listener);
+public boolean removeButtonListener(ButtonName buttonName, OnButtonListener listener);
```

Comment:
The bools / throwing exceptions is necessary here since they don't have NSError values.